### DISCUSSION TITLE
Is this repository up to date?

### DISCUSSION THREAD

**Discussion by roysc** [WGSExtract/WGSExtract-Dev#16] [General]
Is this project still maintained in this repository? I see the latest version on https://wgsextract.github.io/ is 44.8, but the last commit here is months older, and the program version in `latest-release-Dev.json` is 44.4. Where can I find the latest code?

**Comment by RandyHarr** [comment]
No. Each release is the full source code. The JSON file mentioned could not be reliably obtained (with date, etc) programmatically. So is now retrieved, like the source files  packages and installers, from get.wgse.io.  See the release.json file in the installer for the pointer to the latest- release json file you mentioned.

The installer will delete files unneeded for operation by a normal user on a platform. To obtain a true, complete source release, you need to overlay the installer and program packages back onto an installation (after downloading and unzipping). The package pointers are in the latest release json file. Note that the msys2 and cygwin folders with their package json's are only installed on those platforms.  So if you wish those also, you have to merge releases.

Partly due to large binary components to a release, and constant restructuring of folders, we have not found it effective to utilize github in its entirety as of yet. Always open to better ideas.

**Reply by roysc** [reply]
> To obtain a true, complete source release, you need to overlay the installer and program packages back onto an installation

Thanks. This is what I did. I just wanted to pull out enough of the code to run a specific extraction and was having issues with an older release. If I need to work with this further and have time I may refactor it into a simplified CLI script.
